<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="PaintLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>148</y>
     <width>250</width>
     <height>352</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TextButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>230</y>
     <width>120</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	background-color: rgba(139, 140, 158, 150);
	border:1px solid rgb(139, 140, 158);
	font: 20pt &quot;StopSCD&quot;;
}
QPushButton:hover{
	background-color: rgba(139, 140, 158, 200);
}
QPushButton:checked{
	background-color: rgb(139, 140, 158);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="TextButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>320</y>
     <width>120</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	background-color: rgba(139, 140, 158, 150);
	border:1px solid rgb(139, 140, 158);
	font: 20pt &quot;StopSCD&quot;;
}
QPushButton:hover{
	background-color: rgba(139, 140, 158, 200);
}
QPushButton:checked{
	background-color: rgb(139, 140, 158);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="TextButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>410</y>
     <width>120</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	background-color: rgba(139, 140, 158, 150);
	border:1px solid rgb(139, 140, 158);
	font: 20pt &quot;StopSCD&quot;;
}
QPushButton:hover{
	background-color: rgba(139, 140, 158, 200);
}
QPushButton:checked{
	background-color: rgb(139, 140, 158);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_1</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_3</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintLabel</class>
   <extends>QLabel</extends>
   <header>Search</header>
  </customwidget>
  <customwidget>
   <class>TextButton</class>
   <extends>QPushButton</extends>
   <header>Search</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
